Only the analysis of the DNA or RNA of single cells, and not just the DNA or RNA of larger samples (bulks), enables to understand the foundations of nature at the finest resolution.
For example, evolution, the process of varying DNA over time, happens at the level of single cells in the first place.
Regardless of whether DNA modifications are due to environmental (somatic) mutations, DNA replication during cell division or during germ cell formation, the DNA of only single cells is affected.
As for diseases, in particular cancer reflects an (aberrant) evolutionary process that yields a high degree of cell-to-cell heterogeneity.
As another example, consider developmental processes in stem cells, immune cells or organ formation.
Only monitoring single cells will yield sufficient insight to understand symmetry breaking in early embryo development, the differentiation and adaptation of immune cells, or the temporal and spatial formation of organs.
Single cell technologies seek to turn cell-to-cell heterogeneity at the level of DNA or RNA from a problem into a strong advantage.
Therefore, the application of such technologies are bearing the promise to revolutionize large areas of molecular biology; we are probably only beginning to appreciate the large variety of single-cell sequencing applications for fundamental and clinical research.
Recent experimental advances, such as droplet microfluidics, have established breakthroughs that allow to sequence the DNA or RNA of tens of thousands (and not only of hundreds) of single cells in a massively parallel manner.
The consequence is that single cell sequencing is on the verge of entering a new era, Single Cell Data Science.
Ultra-high throughput sequencing protocols establish a clear boost for single cell biology.

This era will also be characterized by computational issues that are intriguing to data and computer scientists, statisticians and modelers.
Since the experimental advances have been very recent, the majority of such challenges are still fresh and yet to overcome.
They will also have a decisive impact on how to store, arrange and what and when to extract from single cell based fragment datasets.
The development of statistical frameworks, computational infrastructure and support are required.

The goal of this workshop is to bring together single cell researchers of various related areas, such as cancer heterogeneity and evolution, immune system and early (stem cell) development, modeling, statistics and computer/data science.
Together, we will review the current status in experimental technologies and future progress to be expected.
We will then pinpoint computational and statistical challenges, and the avenues of research they can open up.
We will condense our analyses into a white paper that summarizes and reviews the status, expected experimental developments, all questions, challenges and possible solutions in single cell data science as well as new avenues of research these developments promise for single cell biologists.
The white paper shall act as a guideline and compendium for future research for all communities contributing to and profiting from Single Cell Data Science.
We are planning to publish this white paper in a journal of high impact, in order to share our insights with all communities involved.

# Organizers
* Johannes Köster, University of Duisburg-Essen, Harvard Medical School, Boston
* Alice McHardy, Helmholtz Center for Infection Research, Braunschweig
* Ben Raphael, Department of Computer Science, Princeton University
* Alexander Schönhuth, Centrum Wiskunde & Informatica, Amsterdam, Department of Biology, Universiteit Utrecht
* Sohrab Shah, Departments of Pathology and Computer Science, University of British Columbia, Vancouver

# Invited speakers

* Niko Beerenwinkel, ETH Zürich
* Hans Clevers, Princess Maxima Center for Pediatric Oncology
* Maria Florescu, Hubrecht Institute Utrecht
* Victor Guryev, ERIBA Groningen
* Stephanie Hicks, Dana-Farber Institute / Harvard University​ 
* Rafael Irizarry, Harvard University Statistics Department
* Katharina Jahn, ETH Zürich
* Jan Korbel, European Moclecular Biology Laboratory (EMBL), Heidelberg 
* Peter Lansdorp, UMC Groningen and BC Cancer Agency, Vancouver
* Prisca Liberali, Friedrich-Miescher Institute, University Basel
* John Marioni, Cancer Research UK, Cambridge
* Tobias Marschall, Max Planck Institute for Informatics
* Roeland Merks, Centrum Wiskunde & Informatica and Department of Mathematics, University Leiden
* Jeffrey Moffitt, Harvard University
* Nick Navin, MD Anderson Cancer Center, Houston, Texas
* Luca Pinello, Massachusetts General Hospital, Harvard Medical School
* Alexandros Stamatakis, Heidelberg Institute for Theoretical Studies
* Oliver Stegle, European Molecular Biology Laboratory - European Bioinformatics Institute, Cambridge
* Ewa Szcureck, University of Warsaw
* Valerie Taly, Paris Decartes
* Sarah Teichmann, Wellcome Trust Sanger Institute, Cambridge, UK
* Fabian Theis, TU München, Germany
* Catalina Vallejos, Alan Turing Institute and Department of Statistical Science, University College London
* Jörg Vogel, U Würzburg, Germany
* Zhijin Wu, Department of Biostatistics, Brown University
* Nir Yoseph, UC Berkeley


# Venue

The workshop will take place at the Oort building of the Lorentz Center in Leiden, Netherlands [[Google Maps]](https://goo.gl/maps/1ZuJ8p5ZjUo).
